examples/manifold/plot_compare_isomaps.py

```diff
@@ -0,0 +1,83 @@
+"""
+=========================================
+ Comparison of Isomap methods
+=========================================
+
+An illustration of dimensionality reduction on the Swiss-roll
+data set with both Isomap and Landmark Isomap (L-Isomap) methods.
+"""
+
+# Author: Lucas David -- <lucasolivdavid@gmail.com>
+# License: BSD 3 clause, (C) 2015
+
+print(__doc__)
+
+from time import time
+import matplotlib.pyplot as plt
+from matplotlib.ticker import NullFormatter
+from mpl_toolkits.mplot3d import Axes3D
+from sklearn import manifold, datasets
+
+# Next line to silence pyflakes. This import is needed.
+Axes3D
+
+isomap_params = {'n_neighbors': 10, 'n_components': 2}
+l_isomap_params = {'n_neighbors': 10, 'n_components': 2, 'landmarks': 'auto'}
+
+comparisons = [
+    (
+        n_samples,
+        (manifold.Isomap, isomap_params),
+        (manifold.Isomap, l_isomap_params),
+    ) for n_samples in range(1000, 16001, 5000)]
```

Comment:
n_samples=16000 might lead to a large memory use, if this code is run by the circleCI build – it would create two 16000x16000 matrices – one in each isomap approach.